integrations/Engage/browser.js

```diff
@@ -62,23 +62,12 @@ class Engage {
     logger.debug("===In Engage identify");
 
     const { message } = rudderElement;
-    let { userId, originalTimestamp } = message;
+    const { userId, fName, number, lName } = getDefinedTraits(message);
```

Comment:
for userId align with cloud mode which is:
 const uid =
    getDestinationExternalID(message, "engageId") ||
    getFieldValueFromMessage(message, "userIdOnly");

Reply:
Okay making a new one in `getDefinedTraits` for `userIdOnly`